project_phase/models/project_project.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class ProjectPhase(models.Model):
```

Comment:
@oihane what do you think about file naming comment? Should I make that refactoring?

Reply:
odoo has named stages (no specified that they are for tasks) and that's why they are named project.phase and not stage. Otherwise putting each object in a different model/file, we do not see any problem.

cc @anajuaristi